internal/controller/workloadmonitor_controller.go

```diff
@@ -124,7 +124,12 @@ func (r *WorkloadMonitorReconciler) reconcileServiceForMonitor(
 		}
 	}
 
-	resources["public-ips"] = q
+	var resourceLabel string
+	if svc.Annotations != nil {
+		resourceLabel = svc.Annotations["metallb.universe.tf/ip-allocated-from-pool"]
```

Comment:
Metallb was moved from `metallb.universe.tf` to `metallb.io` for quite a long ago:

```yaml
apiVersion: metallb.io/v1beta1
kind: IPAddressPool
metadata:
  name: expensive
  namespace: metallb-system
spec:
  addresses:
  - 42.176.25.64/30
  autoAssign: false
```

Reply:
That's fair, but the annotations on the Service objects still look like this:
```
metallb.universe.tf/ip-allocated-from-pool: cozystack-public
```
I'm open to using either labeling scheme.